src/icalendar/prop/conference.py

```diff
@@ -85,14 +85,20 @@ def from_uri(cls, uri: vUri | str):
 
     def to_uri(self) -> vUri:
         """Convert the Conference to a vUri."""
-        params = {}
-        if self.feature:
-            params["FEATURE"] = self.feature
-        if self.label:
-            params["LABEL"] = self.label
-        if self.language:
-            params["LANGUAGE"] = self.language
+        def normalize(value: str | list[str] | None) -> str | None:
+            if value is None:
+               return None
```

Comment:
Indentation inconsistency - line 90 has 15 spaces, should have 16:

```suggestion
                return None
```